### PR TITLE
Card Component

### PR DESCRIPTION
**Pull request by abutaher11**
preview--https://j7nwdu.csb.app/Components/Card/Card.html

added--6 card types

### CODE REVIEW

**Review by Abhijeet43 — APPROVED** [review]
- The Cards are looking fine
- Suggestions: Indentations could be fixed in some files, alt attributes are blank fill them up

**Comment by abutaher11** [comment]
>

**Comment by abutaher11** [comment]
> * Suggestions: Indentations could be fixed in some files, alt attributes are blank fill them up
Thanks for the review, will implement those changes in the next pull

File: Components/Card/Card.css [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+.upperSection:hover {
+    background-color: #f5f5f5;
+    cursor: pointer;
+}
+  
+.basic-card {
+    display: flex;
+    flex-flow: column nowrap;
+    width: 22rem;
+    /* box-shadow: 3px 3px 80px 1px rgba(207, 207, 207, 0.5);   */
```
Comment (AnkurChunekar) [inline Components/Card/Card.css:93]: remove unwanted comments

**Review by AnkurChunekar — APPROVED** [review]

File: Components/Card/Card.css (hunk 1 above)
Comment (abutaher11) [reply Components/Card/Card.css:93]: Thanks for noticing

**Pull request merged**